routers/routes/recovery.go

```diff
@@ -94,8 +93,8 @@ func Recovery() func(next http.Handler) http.Handler {
 
 					w.Header().Set(`X-Frame-Options`, `SAMEORIGIN`)
 
-					if setting.RunMode != "prod" {
-						store.Data["ErrorMsg"] = combinedErr
+					if !setting.IsProd() {
+						store.Data["ErrMsg"] = combinedErr
```

Comment:
I think this is fixed on master and just needs a branch update.
https://github.com/go-gitea/gitea/pull/14332/files#diff-072d0a315b40a7156ab814b33c6626344c8e171eb9bd598c700daf804baf8e4cR98

Reply:
Damn I missed this! Yes this must be changed back.

Reply:
```suggestion
						store.Data["ErrorMsg"] = combinedErr
```